searches/BreathFirstSearch.cpp

```diff
@@ -0,0 +1,95 @@
+#include <stdio.h>
+
+#define stack_size 10000
+
```

Comment:
Please follow this code style conventions https://google.github.io/styleguide/cppguide.html